templates/docs/patterns/navigation/index.md

```diff
@@ -43,6 +43,15 @@ By default, dropdowns are left-aligned with their parent; if you'd prefer the me
 View example of the dropdown pattern
 </a></div>
 
+## Sliding
+
+You can add nested dropdowns to the navigation by nesting one or more `p-navigation__dropdown` in `p-navigation__item--dropdown-toggle` components.
```

Comment:
Just mention the `p-navigation--sliding` class as well.

And I guess nesting is optional, right? You can have sliding navigation (new style) even if you don't have nested dropdowns, likely without dropdowns as well (obviously not much of a point using it then)?

Reply:
Yeah you're right, not much would be sliding then but it's technically possible!

Reply:
And should be, I think at some point we just want this to be default.